src/pages/o/[orgId]/contact.tsx

```diff
@@ -36,9 +39,9 @@ export default function OrgContactPage() : JSX.Element {
     );
 }
 
-OrgContactPage.getLayout = function getLayout(page, props) {
+OrgContactPage.getLayout = function getLayout({ page, props } : LayoutParams) {
```

Comment:
Is this really right? If I'm not mistaken, this function now expects a single argument (an object with two attributes, `page` and `props`) but when invoked from `_app.tsx` two separate arguments are passed, which seems more reasonable to me.

I believe the `LayoutParams` type has been incorrectly defined. To be honest I'm surprised that this is working. Is it working? Or just passing the linter?